stack2nix.cabal

```diff
@@ -1,5 +1,5 @@
 name:                stack2nix
-version:             0.1.3.0
+version:             0.1.3.1
```

Comment:
I think we should bump to 0.2 once we fix the last outstanding bug :)

Reply:
👍